seqr/views/utils/dataset_utils.py

```diff
@@ -401,30 +395,41 @@ def _load_rna_seq(model_cls, file_path, user, mapping_file, ignore_extra_samples
 
             samples_by_id[(sample_id, project)][gene_id] = row_dict
 
+    conflict_messages = []
+    for project, sample in samples_with_conflict_tissues.items():
+        messages = []
+        for sample_id, tissue_types in sample.items():
+            sample_id_to_tissue_type.pop((sample_id, project))
+            samples_by_id.pop((sample_id, project))
+            messages.append(f'{sample_id} ({", ".join(sorted([REVERSE_TISSUE_TYPE[tissue_type] for tissue_type in tissue_types]))})')
+        conflict_messages.append(f'{", ".join(messages)} in the project {project}')
+    num_samples = sum([len(v) for v in samples_with_conflict_tissues.values()])
+    warnings = [f'Skipped loading for the following {num_samples} tissue type unmatched sample(s) in '
+                f'{len(samples_with_conflict_tissues)} project(s): {", ".join(conflict_messages)}'] if num_samples else []
```

Comment:
grouping samples by project is much less readable to the end user than a clear list of samples, like we curretly have in our existing warnings. In the event a sample ID is included multiple times, the user will be smart enough to figure out which one is the issue. Just make this warning look like the warning we show for unmatched samples, per my previous review